docs/user/README.md

```diff
@@ -44,6 +44,12 @@ The `awsnfsbackupschedule.cloud-resources.kyma-project.io` CRD describes the use
 of the `AwsNfsVolume` instances at regular intervals.
 To learn more, read the [AwsNfsBackupSchedule Custom Resource](./resources/04-120-10-aws-nfs-backup-schedule.md)  documentation.
 
+### AwsNfsVolumeRestore CR
+
+The `awsnfsvolumerestore.cloud-resources.kyma-project.io` CRD describes the GCP Filestore
+restore operation.
```

Comment:
```suggestion
```
I don't like splitting sentences between lines

Reply:
I see this happening in the other doc as well; seems to be automatic setting in your editor. I don't love that, but won't comment on it furthermore.